packages/api/src/router/activity.ts

```diff
@@ -0,0 +1,78 @@
+import { z } from "zod";
+
+import { adminProcedure, createTRPCRouter, protectedProcedure } from "../trpc";
+
+export const activityRouter = createTRPCRouter({
+  getSchedule: protectedProcedure
+    .input(z.object({ day: z.date() }))
+    .query(async ({ ctx, input }) => {
+      return await ctx.db.activity.findMany({
+        where: {
+          day: {
+            gte: new Date(input.day),
```

Comment:
do you need to do new Date here?

Reply:
Yea...I am not sure why but it doesn't work if its not there...